docs/sql/statements/create-repository.rst

```diff
@@ -669,6 +669,50 @@ Parameters
 
   The container path to use for snapshots.
 
+.. _sql-create-repo-gcs-compress:
+
+**compress**
+  | *Type:*    ``boolean``
+  | *Default:* ``true``
+
+  Whether CrateDB should compress the metadata part of the snapshot or not.
+
+  CrateDB does not compress the actual table data.
+
+.. _sql-create-repo-gcs-chunk_size:
+
+**chunk_size**
+  | *Type:*    ``bigint`` or ``text``
+  | *Default:* ``null``
+
+  Defines the maximum size of any single file that comprises the snapshot. If
+  set to ``null``, CrateDB will not split big files into smaller chunks.  You
+  can specify the chunk size with units (e.g., ``1g``, ``5m``, or ``9k``). If
+  no unit is specified, the unit defaults to bytes.
+
+.. _sql-create-repo-gcs-connect_timeout:
+
+**connect_timeout**
+  | *Type:*    ``text``
+  | *Default:* ``20 seconds``
```

Comment:
why 20s? At open-search, default is `0` which is documented to use the google sdk's default.

Reply:
The default value from the library is 20s. I thought i make it explicit.